src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotJVMCIBackendFactory.java

```diff
@@ -57,6 +57,7 @@ static <CPUFeatureType extends Enum<CPUFeatureType>> EnumSet<CPUFeatureType> con
                     Class<CPUFeatureType> enumType,
                     Map<String, Long> constants,
                     long features,
+                    long extra_features,
```

Comment:
All other arguments have a javadoc `@param` above, you should probably add one too then ;)

Reply:
Ah, and is it not more "java" to use `extraFeatures`?